### ISSUE TITLE
Error "found invalid field namespace for v1beta1.RoleRef" installing on OS4

### ISSUE THREAD

**Issue by ldimaggi** [operator-framework/operator-marketplace#168]
Seeing this error installing on an OS4 cluster:

```
kubectl apply -f deploy/upstream/
namespace "marketplace" created
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
customresourcedefinition "catalogsourceconfigs.operators.coreos.com" configured
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
customresourcedefinition "operatorsources.operators.coreos.com" configured
serviceaccount "marketplace-operator" created
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
clusterrole "marketplace-operator" configured
role "marketplace-operator" created
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
clusterrolebinding "marketplace-operator" configured
deployment "marketplace-operator" created
error: error validating "deploy/upstream/06_role_binding.yaml": error validating 
data: found invalid field namespace for v1beta1.RoleRef; if you choose to ignore 
these errors, turn validation off with --validate=false

```

**Comment by SamiSousa** [comment]
This is being addressed in #144

**Comment by aravindhp** [comment]
Fixed by #144
